pingpong/ud_pingpong.c

```diff
@@ -57,7 +57,7 @@ static void *payload;
 static size_t buffer_size;
 static size_t prefix_len;
 static size_t max_msg_size = 0;
-static int timeout = 5;
+static int timeout = 30;
```

Comment:
It would be better to have 2 different timeout values.  One for the initial setup, and another for the message timeout.  A message timeout of 5 seconds should be more than enough.  After setup has occurred, if the timeout value is still the default, it should be reduced back down to 5 seconds.